### PR TITLE
[contrib/telega-bridge-bot] support latest telega version.

### PR DESCRIPTION
**Pull request by BlindingDark**
- support accent colors
- fallback to download original image if not support thumbnail

### CODE REVIEW

File: contrib/telega-bridge-bot.el [hunk 1]
```diff
@@ -237,9 +240,18 @@ If FORCE-UPDATE is non-nil, force update the file."
              (time-subtract (current-time) telega-bridge-bot-matrix-avatar-expires))))
       (delete-file file-path))
     (telega-bridge-bot--download-async
-     url
+     thumbnail-url
      file-path
-     (lambda () (telega-bridge-bot--download-async-callback chat-id msg-id)))))
+     (lambda ()
+       (if (not (zerop (nth 7 (file-attributes file-path))))
```
Comment (zevlg) [inline contrib/telega-bridge-bot.el:246]: Use `file-attribute-size` instead of `(nth 7 ..)`

File: contrib/telega-bridge-bot.el [hunk 2]
```diff
@@ -481,7 +497,7 @@ Will update CHAT-ID MSG-ID when download completed."
 
 (defun telega-bridge-bot--update-msg (msg &rest _)
   "Replace the sender in MSG with the other party's sender."
-  (when-let ((no-modify? (not (telega--tl-get msg :telega-bridge-bot-modified))))
+  (when-let* ((no-modify? (not (telega--tl-get msg :telega-bridge-bot-modified))))
```
Comment (zevlg) [inline contrib/telega-bridge-bot.el:500]: Why this change?

**Review by zevlg — CHANGES_REQUESTED** [review]

File: contrib/telega-bridge-bot.el (hunk 2 above)
Comment (BlindingDark) [reply contrib/telega-bridge-bot.el:500]: Because I got a hint that it will be obsoleted in the latest Emacs version.
https://github.com/emacs-mirror/emacs/blame/6a07dc19ac65eaa3ee1eeb814af85bde7fd2c509/etc/NEWS#L832

File: contrib/telega-bridge-bot.el (hunk 1 above)
Comment (BlindingDark) [reply contrib/telega-bridge-bot.el]: Thanks! Updated.

**Review by zevlg — APPROVED** [review]

**Comment by zevlg** [comment]
Merged in, thanks!
